Week7Threading/Week7Threading/PrimeCalculator.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+namespace Week7Threading
+{
+    /// <summary>
+    /// This class contains the methods and associated code used to calculate prime numbers.
+    /// </summary>
+    public class PrimeCalculator
+    {
+        /// <summary>
+        /// This method calculates if a number is prime using a sequential for loop.
+        /// </summary>
+        /// <param name="userNumber"></param>
```

Comment:
Parameters and return values should be documented as well, describing their purpose. (-0.5)